test/functional/gridfs.test.js

```diff
@@ -3884,4 +3884,37 @@ describe('GridFS', function() {
       });
     }
   });
+
+  /**
+   * @ignore
+   */
+  it('should correctly create index', {
+    metadata: {
+      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
+    },
+
+    // The actual test we wish to run
+    test: function(done) {
+      var configuration = this.configuration;
+      var GridStore = configuration.require.GridStore;
+      var client = configuration.newClient(configuration.writeConcernMax(), { poolSize: 1 });
+
+      client.connect(function(err, client) {
```

Comment:
nit: there is a mix of big arrow notation and non in this file. I usually wouldn't comment, but I think the fact that we so often copy a whole block of code for testing indicates we should focus on these stylistic details, they are likely to persist otherwise